api/src/test/java/com/epam/pipeline/manager/docker/DockerContainerOperationManagerTest.java

```diff
@@ -76,6 +78,7 @@
     private static final long REGION_ID = 1L;
     private static final Long RUN_ID = 2L;
     private static final int CANNOT_EXECUTE_EXIT_CODE = 126;
+    private static final int PAUSE_TIMEOUT = 86400;
 
     @InjectMocks
     @Spy
```

Comment:
Do we still need `Autowired` here?

Reply:
You are absolutely right. This `Autowired ` annotation not necessary in this class.